src/Discord/Parts/Channel/StageInstance.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+/*
+ * This file is a part of the DiscordPHP project.
+ *
+ * Copyright (c) 2015-present David Cole <david.cole1340@gmail.com>
+ *
+ * This file is subject to the MIT license that is bundled
+ * with this source code in the LICENSE.md file.
+ */
+
+namespace Discord\Parts\Channel;
+
+use Discord\Parts\Guild\Guild;
+use Discord\Parts\Part;
+
+/**
+ * A Stage Instance holds information about a live stage. on a Discord guild.
+ *
+ * @property string  $id                    The unique identifier of the Stage Instance.
+ * @property string  $guild_id              The unique identifier of the guild that the stage instance associated to.
+ * @property Guild   $guild                 The guild that the stage instance associated to.
```

Comment:
`Guild|null`? Not sure if the guild is nullable here but on `getGuildAttribute()` it has `?Guild`